requirements/requirements.txt

```diff
@@ -0,0 +1,9 @@
+phoneme_guesser~=0.1
+mycroft-messagebus-client~=0.9.1,!=0.9.2,!=0.9.3
+inflection~=0.5
+pexpect~=4.8
+requests~=2.26
+json_database~=0.5
+kthread~=0.2
+pyxdg~=0.27
```

Comment:
This was supposed to be swapped for xdg and some manual code because LGPL

Reply:
I think this is the package spec'd in ovos-core and mycroft-core.. refactoring this should be fine, but just a note that it adds another dependency to most installations. I think the refactor will also require some code changes and thus be out of scope for this PR

Reply:
That's probably worth escalating, considering.